src/types.ts

```diff
@@ -19,6 +19,11 @@ export type LoadLocaleOptions = FilepathOrSource & {
   bundleOptions?: FluentBundleOptions;
 };
 
+export type NestedTranslation = {
+  belongsTo: LocaleId;
+  translationSource: string;
+};
```

Comment:
Let's stay consistent with the existing code and do
```suggestion
export interface NestedTranslation {
  belongsTo: LocaleId;
  translationSource: string;
}
```